my-app/src/pages/HeadCube/TopUsers/TopUsers.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FC } from 'react';
+import User from './User/User';
+import clN from './topUsers.module.scss';
+
+// eslint-disable-next-line react/function-component-definition
+const TopUsers: FC = () => {
+	return (
+		<div className={clN.topUsers}>
+			<User />
+			<User />
+			<User />
+			<User />
```

Comment:
use array and map for rendering similar components

Reply:
++